content/blog/productivity-porn.md

```diff
@@ -0,0 +1,36 @@
+- **An increasingly polarized, misinformed, and enraged outlook on life** as the incentives of our digital world push you deeper into a filter bubble. In the extreme case even leading to radicalization.
+- **A crippling loss of time and agency** caused by the addictive and distracting nature of these platforms. The push notification acting as a phantasmal chain tethering us to our phone. Watching content not because you truly want to but just --- because?
+
+But, what I've recently come to realize is that there is an even more pernicious way in which social media is negatively impacting my life. Social media has become my productivity porn.
+
+Productivity porn is anything that after having been consumed makes you feel like you were productive when in reality you didn't actually do anything. Just like regular pornography it stimulates you without even performing the act. It is a hollow imitation of the real thing.
+
+Examples of productivity porn include but are not limited to: reading a tweet by a top VC about how to become a better startup founder; watching a Youtube video about the 7 mistakes you need to avoid at the gym; perusing a Hacker News thread about how to improve the code you write. All of these activities deceptively make you feel like you've done something productive. "I just learned something new", you tell yourself. And well this is true, you never actually did the thing you were setting out to do in the first place. A viscious cycle forms where you spend all your time thinking about doing things instead of actually doing them. As this cycle continues your productivity approaches zero.
+
+Keep in mind that productivity is a personal thing. It can look like a variety of things: founding a startup, working out at the gym, building a side project, reading a book, growing a garden. Regardless, the key is that you benefit from doing the actual activity, but not from thinking about doing it. Hence the problem with productivity porn.
+
+I only realized I had this addiction when it was pointed out to me over and over again. [Start Marketing The Day You Start Coding](https://robwalling.com/assets/ebook.pdf) by Rob Walling, [The drawing advice that changed my life](https://www.youtube.com/watch?v=M6NsEDwHHiE&t=512s&ab_channel=struthless) by struthless, [How to get famous on twitter](https://podcasts.apple.com/us/podcast/how-to-get-famous-on-twitter-and-other-listener-questions/id1469759170?i=1000558020601) by the My First Million podcast, and [The Cooking Skill That Can't Be Taught](https://www.youtube.com/watch?v=gYwkKaK5yeQ&ab_channel=InternetShaquille) by Internet Shaquille all were telling me the message I needed to hear. They each said it in a different way, but the core of their message was the same. Stop thinking (reading, listening, watching etc.) about how to do something and just go do it. Or using my terminology, they were all saying that I was addicted to productivity porn[^1].

```

Comment:
```suggestion
I only realized I had this addiction when it was pointed out to me over and over again. [Start Marketing The Day You Start Coding](https://robwalling.com/assets/ebook.pdf) by Rob Walling, [The drawing advice that changed my life](https://www.youtube.com/watch?v=M6NsEDwHHiE&t=512s&ab_channel=struthless) by struthless, [How to get famous on twitter](https://podcasts.apple.com/us/podcast/how-to-get-famous-on-twitter-and-other-listener-questions/id1469759170?i=1000558020601) by the My First Million podcast, and [The Cooking Skill That Can't Be Taught](https://www.youtube.com/watch?v=gYwkKaK5yeQ&ab_channel=InternetShaquille) by Internet Shaquille all were telling me the message I needed to hear. They each said it in a different way, but the core of their message was the same. Stop thinking (reading, listening, watching, etc.) about how to do something and just go do it. Or using my terminology, they were all saying that I was addicted to productivity porn[^1].

```
Note on that last sentence: I don't think they were necessarily saying that you were addicted to productivity porn. Their core message seems to be that productivity porn is not a useful way to accomplish the things you set out to do, which in turn made you realize that you were addicted to productivity porn. Does that difference make sense? I don't feel like I really explained that well